docs/bcd-workflow.md

```diff
@@ -0,0 +1,34 @@
+# BCD workflow
```

Comment:
If this is specifically about github **workflows** (as in, the things that run in the **Actions** tab of the github UI), then I suggest adding an introduction that says so. Workflow is, otherwise, a pretty generic term. I read pretty far down the doc before realizing this might be about GH workflows.

Reply:
Very partial contribution here ahead of a full review:

It would be nice if the beginning of this doc communicated expectations better:

- Who this is for, the audience
- What it's about (covered by the section below)
- When you'd need the information (e.g., "Read this document to decide/fix/change/document…" as the case may be).